### PR TITLE
Leaderboard setup

### PR DESCRIPTION
**Pull request by brhanuhailu**
### In this pull request I set up a JavaScript project for the Leaderboard list app, using:

- [X] npm project with webpack.
- [X] ES6 features.
- [X] Notably modules.
- [X] GitFlow.
- [X] Linter error checked

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,217 @@
...
+
+we would like to thank Microverse organization
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```
Comment (Ghiftee) [inline README.md:212]: - [ ] Kindly add a license to your README as it defines the terms by which your app might be used and it gives your project more professionalism.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (Ghiftee) [inline package.json]: - [ ] Your app need to have a way to start using ```npm scripts``` as it is an ```npm project with webpack```. Kindly ensure that this is possible.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @brhanuhailu  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ You used Gitflow
✔️ App matches the given wireframe
✔️ No linter errors

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @brhanuhailu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- PR is professional :+1: 
- Great app UI :+1: 
- Webpack is used to bundle the app :+1: 
- Descriptive README.md file :+1: 
- Linter checks passed :heavy_check_mark: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**